src/component/MainContent.jsx

```diff
@@ -1,12 +1,19 @@
 import React from 'react';
 import classes from './MainContent.module.css'
-
+import Input from './UI/Input'
+import ButtonEquals from "./ButtonEquals";
 
 const MainContent = () => {
   return (
     <div className={classes.Main__content}>
       <hr className={classes.Hr__line}/>
-      <h1 className={classes.Main__title}>Maks</h1>
+      <Input/>
+      <span>+</span>
+      <Input/>
+      <span>=</span>
+      <ButtonEquals/>
+      <ol className={classes.Main__list}></ol>
+      <h1 className={classes.Main__title}>Dramatic</h1>
```

Comment:
in this place should be new component, which will take an array of calculation results as props
`<ResultTable items={items} />`

items can look like:
```
items = [
{
  id: 'some_id'
 result: 10,
 date: 27.08.2022
}, ...
]
```

Best practice is to store items in state of component, check this link:
https://reactjs.org/docs/hooks-reference.html#functional-updates

and try to create `useState` hook

Finally, every time you click button sum, it will update your state ==> will insert new item element into the items list